scipy/stats/tests/test_stats.py

```diff
@@ -1875,65 +1875,95 @@ def test_zscore_nan_raise(self):
 
         assert_raises(ValueError, stats.zscore, x, nan_policy='raise')
 
-    def test_mad(self):
-        dat = np.array([2.20, 2.20, 2.4, 2.4, 2.5, 2.7, 2.8, 2.9, 3.03,
+
+class TestMedianAbsDeviation(object):
+    def setup_class(self):
+        self.dat_nan = np.array([2.20, 2.20, 2.4, 2.4, 2.5, 2.7, 2.8, 2.9, 3.03,
+                3.03, 3.10, 3.37, 3.4, 3.4, 3.4, 3.5, 3.6, 3.7, 3.7,
+                3.7, 3.7, 3.77, 5.28, np.nan])
+        self.dat = np.array([2.20, 2.20, 2.4, 2.4, 2.5, 2.7, 2.8, 2.9, 3.03,
                 3.03, 3.10, 3.37, 3.4, 3.4, 3.4, 3.5, 3.6, 3.7, 3.7,
-                3.7, 3.7,3.77, 5.28, 28.95])
-        assert_almost_equal(stats.median_absolute_deviation(dat, axis=None), 0.526323)
+                3.7, 3.7, 3.77, 5.28, 28.95])
 
-        dat = dat.reshape(6, 4)
-        mad = stats.median_absolute_deviation(dat, axis=0)
-        mad_expected = np.asarray([0.644931, 0.7413, 0.66717, 0.59304])
+    def test_median_abs_deviation(self):
+        assert_almost_equal(stats.median_abs_deviation(self.dat, axis=None), 0.355)
```

Comment:
Line length:
```suggestion
        assert_almost_equal(stats.median_abs_deviation(self.dat, axis=None),
                            0.355)
```
I realize this was already too long before being changed, but when modifying existing code, we might as well also fix PEP-8 issues.